File: problems/71/Main.py
#--SPLIT--

import math

#--SPLIT--
#-class Solution:
    # @param A : string
    # @return an integer
#-    def titleToNumber(self, a):


#--SPLIT--
class Solution:
    def titleToNumber(self, a):
        result = 0
        for i, c in enumerate(a):
            c = a[len(a) - i - 1]
            result += (ord(c) - ord('A') + 1) * 26**i
        return result

#--SPLIT--

if __name__ == "__main__":
    test_cases = int(raw_input())
    for i in range(test_cases):
        a = raw_input()
        s = Solution()
        print(s.titleToNumber(a))

File: problems/71/Main.java
import java.io.*;
import java.util.*;
import java.lang.*;

//--SPLIT--

//-public class Solution {
//-	public int titleToNumber(String a) {
//-	}
//-}

//--SPLIT--
class Solution {
	public int titleToNumber(String a) {
		int result = 0;
		for (int i = 0; i < a.length(); i++) {
			char c = a.charAt(a.length() - i - 1);
			result += (int)c - 'A' + 1) * Math.pow(26, i);
		}
		return result;
	}
}

//--SPLIT--
public class Main {
	public static void main(String[] args) throws Exception {
		BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
		int cases = Integer.parseInt(reader.readLine());
		Solution s = new Solution();
		for (int i = 1; i <= cases; i++) {
			String input = reader.readLine();
			System.out.println(s.titleToNumber(input));
		}
	}
}
